### PR TITLE
Add Thorma90 Network

### PR DESCRIPTION
**Pull request by EinfachFloTV**

### CODE REVIEW

File: minecraft_servers/thorma90/background.png
Comment (DoJapHD) [inline minecraft_servers/thorma90/background.png:1]: Compress image

File: minecraft_servers/thorma90/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/thorma90/background@2x.png:1]: Compress image

File: minecraft_servers/thorma90/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "direct_ip": "thorma90.net",
+    "server_wildcards": [
+      "%.thorma90.net"
+    ],
+    "supported_languages": [
+       "de"
+    ],
+    "social": {
+      "discord": "https://discord.gg/yRZEFzn5Kk",
+      "web": "https://thorma90.net",
```
Comment (DoJapHD) [inline minecraft_servers/thorma90/manifest.json:13]: https://i.imgur.com/RJgActK.png

File: minecraft_servers/thorma90/manifest.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "thorma90",
+    "nice_name": "Thorma90 Netzwerk",
+    "direct_ip": "thorma90.net",
+    "server_wildcards": [
+      "%.thorma90.net"
```
Comment (DoJapHD) [inline minecraft_servers/thorma90/manifest.json:6]: Not a real wildcard
https://i.imgur.com/cEteI6A.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by EinfachFloTV** [comment]
Updated

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**